demonstrations/tutorial_qubit_tapering.py

```diff
@@ -0,0 +1,274 @@
+r"""
+
+Qubit tapering
+==============
+
+.. meta::
+    :property="og:description": Learn how to taper off qubits
+    :property="og:image": https://pennylane.ai/qml/_images/ qubit_tapering.png
+
+.. related::
+    tutorial_quantum_chemistry Quantum chemistry with PennyLane
+    tutorial_vqe A brief overview of VQE
+    tutorial_givens_rotations Givens rotations for quantum chemistry
+    tutorial_adaptive_circuits Adaptive circuits for quantum chemistry
+
+
+*Author: PennyLane dev team. Posted:  2021. Last updated: XX January 2022*
+
+The performance of variational quantum algorithms is considerably limited by the number of qubits
+required to represent wave functions. In the context of quantum chemistry, this
+limitation hinders the treatment of large molecules with algorithms such as the variational quantum
+eigensolver (VQE). Several approaches have been developed to reduce the qubit requirements for
+quantum chemistry calculations. In this tutorial, we demonstrate the symmetry-based qubit
+tapering approach which allows reducing the number of qubits required to perform molecular quantum
+simulations based on the :math:`\mathbb{Z}_2` symmetries present in molecular Hamiltonians
+[#bravyi2017]_ [#setia2019]_.
+
+A molecular Hamiltonian in the qubit basis can be expressed as a linear combination of Pauli words
+as
+
+.. math:: H = \sum_{i=1}^r h_i P_i
+
+where :math:`h_i` is a real coefficient and :math:`P_i` is a tensor product of Pauli and
+Identity operators acting on M qubits

```

Comment:
```suggestion
identity operators acting on M qubits
```
here and elsewhere, there is no need to capitalize identity